### PR TITLE
Attachment with tag container, replaced button from material ui with design system

### PR DESCRIPTION
**Pull request by lassopicasso**
## Description
Encountered that the delete button in container "upload attachment with tag" was too large compared to other elements in the container.

Replaced the button (material ui) and the icon (font awesome) with those from our [design system](https://digdir.github.io/designsystem/?path=/docs/komponentoversikt--page) and [Aksel - NAV  ](https://aksel.nav.no/).
While I was on it, also replaced the check mark icon with the one from Aksel NAV.

Before:
old-example-of-delete-button

After:
image

## Related Issue(s)

- closes #{issue number}

## Verification/QA

- Manual functionality testing
  - [x] I have tested these changes manually
  - [ ] Creator of the original issue (or service owner) has been contacted for manual testing (or will be contacted when released in alpha)
  - [ ] No testing done/necessary
- Automated tests
  - [ ] Unit test(s) have been added/updated
  - [ ] Cypress E2E test(s) have been added/updated
  - [x] No automatic tests are needed here (no functional changes/additions)
  - [ ] I want someone to help me make some tests
- UU/WCAG ([follow these guidelines](https://aksel.nav.no/god-praksis/artikler/utvikling?tema=universell-utforming) until we have our own)
  - [x] I have tested with a screen reader/keyboard navigation/automated wcag validator
  - [ ] No testing done/necessary (no DOM/visual changes)
  - [ ] I want someone to help me perform accessibility testing
- User documentation @ [altinn-studio-docs](https://github.com/Altinn/altinn-studio-docs)
  - [ ] Has been added/updated
  
  - [x] No functionality has been changed/added, so no documentation is needed
  - [ ] I will do that later/have created an issue
  
- Changes/additions to component properties
  - [ ] Changes are reflected in both `src/layout/layout.d.ts` and `layout.schema.v1.json`, and these are all backwards-compatible
  - [x] No changes made
- Support in Altinn Studio
  - [ ] Issue(s) created for support in Studio
  
  - [x] This change/feature does not require any changes to Altinn Studio
- Sprint board
  - [x] The original issue (or this PR itself) has been added to the Team Apps project and to the current sprint board
  - [ ] I don't have permissions to do that, please help me out
- Labels
  - [x] I have added a `kind/*` label to this PR for proper release notes grouping
  - [ ] I don't have permissions to add labels, please help me out

### CODE REVIEW

File: src/layout/FileUploadWithTag/EditWindowComponent.tsx [hunk 1]
```diff
@@ -99,17 +95,17 @@ export function EditWindowComponent(props: EditWindowProps): JSX.Element {
...
                 {!props.mobileView
                   ? getLanguageFromKey('form_filler.file_uploader_list_status_done', props.language)
                   : undefined}
-                <i
-                  className='ai ai-check-circle'
+                <CheckmarkCircleFillIcon
                   role='img'
                   aria-hidden={!props.mobileView}
                   aria-label={getLanguageFromKey('form_filler.file_uploader_list_status_done', props.language)}
+                  style={{ marginLeft: '0.2rem', fontSize: '1.3rem' }}
```
Comment (Magnusrm) [inline src/layout/FileUploadWithTag/EditWindowComponent.tsx:108]: Same as comment above for this line 😊

**Review by Magnusrm — COMMENTED** [review]
Looks good! 🎉 Just some minor nit-picky comments below:

**Comment by lassopicasso** [comment]
Thank you, I have moved the inline styling to a css-module file. :)

**Review by Magnusrm — APPROVED** [review]
Nice work! 🥳

**Pull request merged**